From 45914d5301d694730965872962fc71b9d11f9b3d Mon Sep 17 00:00:00 2001
From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Sat, 31 May 2014 21:36:23 +0200
Subject: [PATCH] util: ignore_file should not allow files ending with '~'

ignore_file currently allows any file ending with '~' while it
seems that the opposite was intended:
a228a22fda4faa9ecb7c5a5e499980c8ae5d2a08

(cherry-picked from 93f1a06374e335e8508d89e1bdaadf45be6ab777)

Resolves: #1147524
---
 src/shared/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/util.c b/src/shared/util.c
index 9f25b15..1590c3a 100644
--- a/src/shared/util.c
+++ b/src/shared/util.c
@@ -1502,7 +1502,7 @@ bool ignore_file(const char *filename) {
         assert(filename);
 
         if (endswith(filename, "~"))
-                return false;
+                return true;
 
         return ignore_file_allow_backup(filename);
 }
